.gitignore

```diff
@@ -31,7 +31,6 @@ custom-elements.json
 packages/*/src/**/*.css.js
 packages/*/custom-elements.json
 packages/**/*.js
-packages/**/spectrum-vars.json
```

Comment:
This is a legacy asset from the vars and expressvars days. Might be worth looking into copying over the metadata.json functionality from CSS though.